docs/standard/microservices-architecture/architect-microservice-container-applications/asynchronous-message-based-communication.md

```diff
@@ -27,7 +27,7 @@ Single-receiver message-based communication is especially well suited for sendin
 
 Once you start sending message-based communication (either with commands or events), you should avoid mixing message-based communication with synchronous HTTP communication.
 
-![](./media/image18.PNG)
+![A single microservice receiving an asynchronous message](./media/image18.png)
```

Comment:
Hi @mairaw, all images in the book are named like this, would it be OK to only extend the post description and leave the image names for some next revision? 

It'll help to get this one out faster.

Reply:
Yes please, we need to move forward fast. We can improve things "on the go" but if it is something that the whole book has in one way, like image files numbering, we cannot stop to do that because of lack of time bandwidth.